src/framesummary/FrameWatchConfigurationViewProvider.js

```diff
@@ -18,30 +21,39 @@ export default class FrameWatchConfigurationViewProvider {
     }
 
     view(selection) {
-        let component;
-        let domainObject = selection[0][0].context.item;
-        const tableConfiguration = new FrameWatchTableConfiguration(domainObject, openmct, this.type);
+        const domainObject = selection[0][0].context.item;
+        const tableConfiguration = new FrameWatchTableConfiguration(domainObject, this.openmct, this.type);
```

Comment:
These one's are working fine with the `this.openmct`?

Reply:
i think i was just generally refactoring and i must have changed this when i changed the `let` to `const`. i reverted this change for now.